### ISSUE TITLE
Error with ppc versions

### ISSUE THREAD

**Issue by chrislovecnm** [bazelbuild/rules_python#1326]
We are getting

e.g.:
```
bazel query "deps(@python_3_10//:python_headers)"
INFO: Repository rules_python~0.24.0~python~python_3_10_x86_64-unknown-linux-gnu instantiated at:
  callstack not available
Repository rule python_repository defined at:
  /private/var/tmp/_bazel_melliot/84bf3287741d1703ed98e552f5dae756/external/rules_python~0.24.0/python/repositories.bzl:381:36: in <toplevel>
INFO: Repository rules_python~0.24.0~python~python_3_10_x86_64-pc-windows-msvc instantiated at:
  callstack not available
Repository rule python_repository defined at:
  /private/var/tmp/_bazel_melliot/84bf3287741d1703ed98e552f5dae756/external/rules_python~0.24.0/python/repositories.bzl:381:36: in <toplevel>
INFO: Repository rules_python~0.24.0~python~python_3_10_aarch64-apple-darwin instantiated at:
  callstack not available
Repository rule python_repository defined at:
  /private/var/tmp/_bazel_melliot/84bf3287741d1703ed98e552f5dae756/external/rules_python~0.24.0/python/repositories.bzl:381:36: in <toplevel>
INFO: Repository rules_python~0.24.0~python~python_3_10_aarch64-unknown-linux-gnu instantiated at:
  callstack not available
Repository rule python_repository defined at:
  /private/var/tmp/_bazel_melliot/84bf3287741d1703ed98e552f5dae756/external/rules_python~0.24.0/python/repositories.bzl:381:36: in <toplevel>
ERROR: Evaluation of query "deps(@python_3_10//:python_headers)" failed: preloading transitive closure failed: no such package '@[unknown repo 'python_3_10_ppc64le-unknown-linux-gnu' requested from @rules_python~0.24.0~python~python_3_10]//': The repository '@[unknown repo 'python_3_10_ppc64le-unknown-linux-gnu' requested from @rules_python~0.24.0~python~python_3_10]' could not be resolved: No repository visible as '@python_3_10_ppc64le-unknown-linux-gnu' from repository '@rules_python~0.24.0~python~python_3_10'

```

The problem is that we don’t have ppc in every version of each python version. We are using PLATFORMS.keys() and hitting cases were we don’t have ppc.

**Comment by chrislovecnm** [comment]
We need to change this https://github.com/bazelbuild/rules_python/blob/49d2b7aadb084ac7cae48583c38af6da2ce41a02/python/private/toolchains_repo.bzl#L155

See:

https://bazelbuild.slack.com/archives/CA306CEV6/p1689608714375079

**Comment by aignas** [comment]
Fixed by #1473.
